resources/test/conftest.py

```diff
@@ -58,6 +69,27 @@ def repr_failure(self, excinfo):
         return pytest.Collector.repr_failure(self, excinfo)
 
     def runtest(self):
+        if self.type == 'unit':
+            self._run_unit_test()
+        elif self.type == 'functional':
+            self._run_functional_test()
```

Comment:
```python
else:
    raise NotImplementedError
```

Reply:
Will do.

I didn't know that Python instantiates classes when they are a raised value. I'm trying to think of when that would be necessary.